src/components/helpers.tsx

```diff
@@ -0,0 +1,12 @@
+import { css } from "styled-components"
+
+export const block = (margin: number = 0) => {
```

Comment:
This could do with some documentation. Use jsdoc so the IDE will nicely display it inline.